iroha-cli/impl/grpc_response_handler.cpp

```diff
@@ -41,12 +41,15 @@ namespace iroha_cli {
     handler_map_[DATA_LOSS] = "DATA_LOSS";
   }
 
-  void GrpcResponseHandler::handle(
+    bool GrpcResponseHandler::handle(
       CliClient::Response<CliClient::TxStatus> response) {
     if (response.status.ok()) {
-      tx_handler_.handle(response.answer);
+        tx_handler_.handle(response.answer);
```

Comment:
Please, use clang-format. We use 2 spaces for indentation like here (https://google.github.io/styleguide/cppguide.html).

Reply:
Also it seems that this part is already here (https://github.com/hyperledger/iroha/pull/1318/files).